react-common/components/share/ShareInfo.tsx

```diff
@@ -92,6 +94,19 @@ export const ShareInfo = (props: ShareInfoProps) => {
         else setShareState("share")
     }
 
+    const handlePublishInVscodeClick = async () => {
+        setShareState("publishing");
+        let publishedShareData = await publishAsync(name, thumbnailUri, isAnonymous);
+        setShareData(publishedShareData);
+        if (!publishedShareData?.error) {
+            setShareState("publish");
+
+            pxt.tickEvent(`share.openInVscode`);
+            window.open(vscodeDevUrl + publishedShareData.url.split("/").pop(), "_blank");
+        }
+        else setShareState("share")
```

Comment:
Do we want to notify the user if there's an error?

Reply:
that's handled when we set the sharedata